spring-native-configuration/src/test/java/org/springframework/validated/ValidatedComponentProcessorTest.java

```diff
@@ -0,0 +1,123 @@
+package org.springframework.validated;
```

Comment:
I think it is perhaps also worth tweaking the existing validator sample we have with a bit of code that needs this component processor (maybe its JDKproxy and AOTproxy variants, but particularly the latter) to verify this all works in a native image.

Reply:
I'll try that tomorrow.

Reply:
I've added classes to validator sample. Now I observe the strange thing where I can see AOT proxy classes being generated yet the resulting image complain they are not available :thinking: